pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -47,12 +47,20 @@ def _strip_raw_data(tensor):
     return tensor
 
 
-def _strip_large_initializer_raw_data(onnx_model, large_tensor_threshold):
-    for init in onnx_model.graph.initializer:
+def _strip_large_initializer_raw_data_from_graph(graph, large_tensor_threshold):
+    for init in graph.initializer:
         if _is_stripped(init):
             continue
         if is_large_tensor(init, large_tensor_threshold):
             _strip_raw_data(init)
+    for node in graph.node:
+        for attr in node.attribute:
+            if attr.name == 'subgraph':
```

Comment:
Maybe this is more generic:
```suggestion
            if attr.HasField('g'):
```